hdr_schemata/models/HDRUK/v2_2_1/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .EnrichmentAndLinkage import EnrichmentAndLinkage
 from .Accessibility import Accessibility
 from .Provenance import Provenance
+from .Observations import Observation
 
```

Comment:
can you add `from .MeasuredProperty import MeasuredProperty`?

This is so that in the future we can do `import MeasuredProperty from hdr_schemata.models.HDRUK.v2_2_1`

Reply:
Makes sense, done!